### PR TITLE
Allows user to choose the default colour

### PR DESCRIPTION
**Pull request by cisoun**
Let the user choose the folders default color.
The script asks the user for the color after the style and replaces the symbolic links.

### CODE REVIEW

**Comment by Foggalong** [comment]
Woah, I wasn't expecting this! The code is a lot shorter than what I was implementing too - I'll check the repo out asap :)

File: script.sh [hunk 1]
```diff
@@ -99,7 +99,29 @@ else
     gerror
 fi
 
-cp -rH files/"${style}"/Numix/* "${dir}"/Numix/
+read -p "Which folder colour do you want? " answer
+if [[ "default,blue,green,grey,orange,pink,purple,red,yellow" == *$answer* ]]; then
```
Comment (cisoun) [inline script.sh]: If the user enter something like "grey,orange", then the script will look for files named "grey,orange*" and maybe crash.
If someone has a better idea to secure this, that would be cool.

Comment (Foggalong) [inline script.sh]: I have a plan for this similar to how we do it for style, so don't worry about that :)

File: script.sh [hunk 2]
```diff
@@ -99,7 +99,29 @@ else
...
+cp -rf files/"${style}"/Numix/* "${dir}"/Numix/
+
+CURRENTCOLOUR=`readlink ${dir}/Numix/16x16/places/folder.svg | cut -d '-' -f 1`
+LINKS=`find -L ${dir}/Numix/*/{actions,places} -xtype l`
+for LINK in $LINKS; do
+	NEWLINK=`readlink $LINK`;
+	if [[ $NEWLINK == *"$CURRENTCOLOUR"* ]]; then
+		NEWLINK=${NEWLINK/${CURRENTCOLOUR}/${colour}};
+		rm -rf $LINK;
+		ln -sf $NEWLINK $LINK;
```
Comment (Foggalong) [inline script.sh]: Do you need do to `rm -rf` since that `ln -sf` will force remove the destination anyway?

**Comment by dirtydancing** [comment]
Sweet! I was about to file a question issue as to whether the global folder colour switcher would be implemented in a similar fashion to the folder style switcher, so no need to file that question anymore :)

**Comment by Foggalong** [comment]
Add your comment as a line note in response to [this](https://github.com/numixproject/numix-folders/pull/50#discussion_r29113662), it stops the thread getting too cluttered with multiple discussions going on :)

**Comment by dirtydancing** [comment]
Sorry, did not mean to clutter the thread, and I have no comment on the code itself, so I commented below. Please just remove as you see fit.

File: script.sh (hunk 2 above)
Comment (cisoun) [inline script.sh]: Yeah you may remove that. I think it would work. I left this line in case of troubles.

**Comment by dirtydancing** [comment]
OK, removed my two previous comments, and I will remove this one as well. Great news anyways :+1:

**Comment by Foggalong** [comment]
As far as I can tell, this is good to go! I'll make some modifications and clean it up to be consistent with the rest of the code, but congratulations: your implementations was a lot smarter than what I had in mind :P

**Pull request merged**